### PR TITLE
Extract Channel Links through getChannelInfo

### PR DESCRIPTION
**Pull request by ChunkyProgrammer**
This PR will allow channel links to be extracted ex: a channel's FaceBook link, Instagram, Merch store

### CODE REVIEW

**Review by efb4f5ff-1298-471a-8973-3d47447115dc — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -77,6 +77,10 @@ ytch.getChannelInfo(payload).then((response) => {
    tags: Array[String], // Will return null if none exist
    channelIdType: Number, 
    alertMessage: String, // Will return a response alert message if any (e.g., "This channel does not exist."). Otherwise undefined
+   channelLinks: {
+      primaryLinks: Array[Object],
```
Comment (PikachuEXE) [inline README.md:81]: I think `Object` should be replaced with the possible properties of objects?

File: app/youtube-grabber.js [hunk 2]
```diff
@@ -22,6 +22,29 @@ class YoutubeGrabber {
...
+        }
+      })
+      links.secondaryLinks = links.secondaryLinks.map(x => {
+        return {
+          url: decodeURIComponent(x.navigationEndpoint.urlEndpoint.url.match('&q=(.*)')[1]),
+          icon: x.icon.thumbnails[0].url,
+          title: x.title.simpleText
+        }
+      })
+    }
```
Comment (PikachuEXE) [inline app/youtube-grabber.js:46]: Would be good to separate variables for result and temp storage (for readability or something)
`links` is for result and `channelHeaderLinksData` is temp storage
```suggestion
    const links = {
      primaryLinks: [],
      secondaryLinks: []
    }
    if (typeof headerLinks !== 'undefined') {
      const channelHeaderLinksData = headerLinks.channelHeaderLinksRenderer
      links.primaryLinks = channelHeaderLinksData.primaryLinks.map(x => {
        return {
          url: decodeURIComponent(x.navigationEndpoint.urlEndpoint.url.match('&q=(.*)')[1]),
          icon: x.icon.thumbnails[0].url,
          title: x.title.simpleText
        }
      })
      links.secondaryLinks = channelHeaderLinksData.secondaryLinks.map(x => {
        return {
          url: decodeURIComponent(x.navigationEndpoint.urlEndpoint.url.match('&q=(.*)')[1]),
          icon: x.icon.thumbnails[0].url,
          title: x.title.simpleText
        }
      })
    }
```

**Review by PikachuEXE — CHANGES_REQUESTED** [review]

**Comment by PrestonN** [comment]
Please make the changes that @PikachuEXE mentioned and then I'll be good to merge this.

**Comment by PrestonN** [comment]
Actually never mind, GitHub allows me to commit them with the click of a button.

**Pull request merged**